paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkConnectorOptions.java

```diff
@@ -301,6 +301,13 @@ public class FlinkConnectorOptions {
                     .withDescription(
                             "Sink committer memory to control heap memory of global committer.");
 
+    public static final ConfigOption<Boolean> SINK_FINISH_GENERATAR_TAG =
+            ConfigOptions.key("sink.finish.generatar-tag")
```

Comment:
Can this feature be a part of `tag.automatic-creation`? We can introduce `BATCH` TagCreationMode.
The downside of this is that it cannot be orthogonal to the automatic creation of tags in streaming writing.

Reply:
Yes, we hope to use tags in batch processing scenarios and generate corresponding tags after the current task ends.

Do you mean to add a batch mode in tag.automatic-creation without adding new configuration? If that's the case, I think it's okay

Reply:
yes, `tag.automatic-creation=batch`

Reply:
Please take another look